tests/test_api/test_validation.py

```diff
@@ -479,6 +479,26 @@ def test_check_bounding_box_size():
         validation.check_bounding_box_size(job, None, max_bounds_area=99.9)
 
 
+def test_has_rtc_static_coverage():
```

Comment:
If we're not mocking these then I'd be concerned about the runtime of the test suite when making all of these CMR queries. I'd prefer two test cases (one granule with coverage, one without) so that the test only has to run two queries.

And perhaps would be good to test the `check_rtc_static_coverage` function itself, rather than the helper function.

Reply:
Ran the test locally. It's faster than I thought, though still noticeably slower than the other tests, so still might be nice to reduce the number of assertions.